simphony_ui/couple_CFDDEM.py

```diff
@@ -127,29 +118,16 @@ def run_calc(output_path, mesh_name):
                              (chansize[0], chansize[1], chansize[2]),
                              (0.0, chansize[1], chansize[2])]
 
-            openfoam_file_io.create_quad_mesh(tempfile.mkdtemp(), mesh_name,
-                                              wrapper_OF, corner_points,
+            openfoam_file_io.create_quad_mesh(output_path, mesh_name,
```

Comment:
Strange. There might be a reason why it uses mkdtemp here, like a scratch directory. You need to check if this change is actually ok or might introduce problems (e.g. conflicts in filenames, large files in an inappropriate place)